mtags/src/main/scala-2/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -34,7 +34,7 @@ import scala.meta.internal.metals.FoldingRangeProvider
 import scala.meta.internal.metals.Trees
 import org.eclipse.lsp4j.DocumentSymbol
 import scala.meta.internal.metals.DocumentSymbolProvider
-import org.eclipse.lsp4j.{DocumentOnTypeFormattingParams, TextEdit}
```

Comment:
The import was actually repeated and this only got reported with Scala 2.13.2 :tada: